evalml/automl/automl_search.py

```diff
@@ -585,10 +585,6 @@ def __init__(
         self.random_seed = random_seed
         self.n_jobs = n_jobs
 
-        if not self.plot and self.verbose:
```

Comment:
Plotly moved from being a core-requirement to one of the regular requirements. `self.plot` is only None if plotly isn't installed. Since we no longer test this case, I removed this to make codecov happy (and users that pip install us shouldn't have an issue regardless)